### DISCUSSION TITLE
Next.js Vercel Domain issue

### DISCUSSION THREAD

**Discussion by peraknezevic** [vercel/next.js#69936] [Help]
### Link to the code that reproduces this issue

https://github.com/peraknezevic/marijanacanak

### To Reproduce

1. Deploy app
2. Update content via dashboard

### Current vs. Expected behavior

I created this small next.js website / crud app and I'm hosting it on vercel. The issue is that whenever I update content via the admin section in the app everything gets updated as it should on myappaddress.vercel.app but the changes don't appear on myaddaddress.com. The only way to make the changes visible has been to redeploy the app. Does anyone know what could be the issue here?  Am I missing something in Next.js or is this Vercel / domain host issue?

### Provide environment information

```bash
Operating System:
  Platform: darwin
  Arch: arm64
  Version: Darwin Kernel Version 24.1.0: Fri Aug 16 19:20:28 PDT 2024; root:xnu-11215.40.42~4/RELEASE_ARM64_T8103
  Available memory (MB): 16384
  Available CPU cores: 8
Binaries:
  Node: 21.1.0
  npm: 10.2.0
  Yarn: 1.22.22
  pnpm: 9.5.0
Relevant Packages:
  next: 14.2.7 // There is a newer version (14.2.9) available, upgrade recommended! 
  eslint-config-next: 14.2.7
  react: 18.3.1
  react-dom: 18.3.1
  typescript: 5.5.4
Next.js Config:
  output: N/A
```

### Which area(s) are affected? (Select all that apply)

Developer Experience

### Which stage(s) are affected? (Select all that apply)

Vercel (Deployed)

### Additional context

The same issue with all Next.js 14 releases so far

**Comment by samcx** [comment]
@peraknezevic This seems like a Vercel issue. Did you make sure to add the custom domain and re-deploy after adding it? This question is also great to ask in https://vercel.community!

**Reply by peraknezevic** [reply]
> @peraknezevic This seems like a Vercel issue. Did you make sure to add the custom domain and re-deploy after adding it? This question is also great to ask in https://vercel.community!

Yes, I redeployed it many times since adding the domain. Thanks, I'll try asking at vercel community!
